themes/wc-eh-docs/layouts/partials/footer.html

```diff
@@ -30,10 +30,13 @@ <h4 class="card-title"><a class="summary-title-link"></a></h4>
 
 <script defer src="{{ $js.RelPermalink }}{{ if $assetBusting }}?{{ now.Unix }}{{ end }}" integrity="{{ $js.Data.Integrity }}"></script>
 
-
+{{ if eq hugo.Environment "production" }}
```

Comment:
do we run hugo somewhere with the environment flag set to production?

maybe it would be better to change the order and check if eq hugo.Environment "staging" so that we don't minify the JS files only in that case?